packages/astro/package.json

```diff
@@ -0,0 +1,136 @@
+{
+  "name": "@clerk/astro",
+  "description": "Clerk SDK for Astro",
+  "version": "0.0.0",
+  "type": "module",
+  "private": false,
```

Comment:
You can remove that line

Reply:
Are you referring to  `private: false` ?

Reply:
Yes